pages/common/rarcrack.md

```diff
@@ -0,0 +1,15 @@
+# rarcrack
+
+> Password cracker for rar, zip and 7z archives.
+
+- Brute force the password for an archive (tries to guess the archive type):
+
+`rarcrack {{path/to/file.zip}}`
+
+- Specify the archive type:
+
+`rarcrack --type {{rar|zip|7z}} {{path/to/file.zip}}`
+
+- Use multiple threads:
+
+`rarcrack --threads {{6}}`
```

Comment:
Does this command just use multiple threads to do nothing? Because I don't see a file name or a mention of it reading from `stdin`.

Reply:
Sorry, I forgot to add the filepath:
```suggestion
`rarcrack --threads {{6}} {{path/to/file.zip}}`
```